internal/policy/output.go

```diff
@@ -58,6 +58,28 @@ func (o *Output) SetPolicyCheck(results []output.CheckResult) {
 	o.ExitCode = output.ExitCode(results)
 }
 
+func (o Output) Violations() []string {
+	violations := make([]string, 0, 10)
```

Comment:
This seems like an arbitrary capacity for the slice. Maybe just use `violations := []string{}` and let the compiler figure out what's a good default value?

Reply:
Thank you for the reference link. Very helpful.

As you explained, `10` is not an arbitrary number. It's a good enough guess based on your understanding of how the slice will be used. That's fine. For posterity, it would be nice if there was a comment in the code explaining the reasoning for choosing that value, and also the use of a named variable to better express intent.